### ISSUE TITLE
hotkey to activate?

### ISSUE THREAD

**Issue by bwanaaa** [UbuntuBudgie/window-shuffler#8]
so i run matrix-wrapper and it sets up everything to defaults and shows the grid. How can i bring up the grid with a hotkey?

**Comment by Jacob-Vlijm** [comment]
You mean another one than Ctrl + S?

**Comment by Jacob-Vlijm** [comment]
@bwanaaa please mention what distro/version you are on and how you installed shuffler, does Ctrl + S work? please clarify a bit.

**Comment by bwanaaa** [comment]
ubuntu 18.04 LTS
ctrl-S brings up a search window when the file manager  is open(a window showing any directory) or if i click anywhere on the desktop. if firefox is the active window, it brings up a save dialog. if terminal is on top it does nothing.

**Comment by bwanaaa** [comment]
Installed from here
https://github.com/UbuntuBudgie/window-shuffler
I put the main folder in /usr/local.
I put the icons in the pixmaps folder as instructed
I ran matrix-wrapper

Now to activate it I have to type into terminal
```

$cd /usr/local/window-shuffler/
$./window-shuffler
```

**Comment by Jacob-Vlijm** [comment]
@bwanaaa Aaaargh, I made quite a typo... I meant Super + S, but that probably isn't set if you're not on Budgie. Please set `/usr/local/windowshuffler/matrix_wrapper` to a shortcut (if you store `matrix_wrapper` anywhere else, set the path accordingly), and make sure  `matrix_wrapper` is executable. Probably simply the shortcut Super + S is available.

**Comment by bwanaaa** [comment]
Can you tell me the best way to do that?

**Comment by fossfreedom** [comment]
Look at settings - keyboard

Scroll to the bottom and add a custom keyboard shortcut. The command is what Jacob stated above.

**Comment by bwanaaa** [comment]
Sorry if I sound stupid, but I did this
I put /usr/local/window_shuffler on my path in my .bashrc file
Now I can type 'window_shuffler' at any terminal prompt and get it to work
However, the shortcut I tried ctrl+alt+S does nothingI tried various commands in the 'custom add shortcut' dialog includingwindow_shuffler./window_shuffler/usr/local/window-shuffler/window_shuffler

I tried other shortcyt combos toosuper+shift+S
nothing worked.

Have you gotten it to work in ubuntu 18.04LTS?



    On Sunday, May 5, 2019, 4:57:23 PM EDT, David Mohammed <notifications@github.com> wrote:  
 
 
Look at settings - keyboard

Scroll to the bottom and add a custom keyboard shortcut. The command is what Jacob stated above.
